Makefile

```diff
@@ -46,6 +42,9 @@ all: manager
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: generate fmt vet manifests
 	mkdir -p ${ENVTEST_ASSETS_DIR}
+	mkdir -p build/config/crds
+	$(KUSTOMIZE) build config/crd > build/config/crds/medusa-operator-crds.yaml
+	$(KUSTOMIZE) build test/config/cass-operator/crd > build/config/crds/cass-operator-crds.yaml
 	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
 	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./controllers/... ./pkg/... -coverprofile cover.out
```

Comment:
this is just a nit, but it might be better to use `&&` instead of `;` between the commands here. Using the semicolon means all of the commands will run, regardless of the success of previous commands. The double ampersand will exit on the first command that fails. So something like
```sh
source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh && fetch_envtest_tools $(ENVTEST_ASSETS_DIR) && setup_envtest_env $(ENVTEST_ASSETS_DIR) && go test ./controllers/... ./pkg/... -coverprofile cover.out
```

Again, this is just a tiny nitpicky thing.

Reply:
good call!